src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,119 @@
+package duke;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * The TaskList class is in charge of the different arrays
+ * indicating the actions, whether it is done, the type, and the counter.
+ */
+public class TaskList {
+    public static String[] actions;
+    public static boolean[] isDone;
+    public static String[] type;
+    public static int counter;
+    public static String[] dueString = new String[100];
+    public static LocalDateTime[] startTime = new LocalDateTime[100];
+    public  static LocalDateTime[] endTime= new LocalDateTime[100];
+
+
+    /**
+     * Constructor of the TaskList
+     * @param actions Array of actions.
+     * @param isDone Array of whether the action is done.
+     * @param type Array of the different type of actions.
+     * @param counter Number of action in tasklist.
+     */
+    public TaskList(String[] actions, boolean[] isDone, String[] type, int counter){
+        TaskList.actions = actions;
+        TaskList.isDone = isDone;
+        TaskList.type = type;
+        TaskList.counter = counter;
+    }
+
+    /**
+     * Formats and saves the to do type of action.
+     * @param action String of name of action.
+     */
+
+    public static void todo(String action) {
+        isDone[counter] = false;
+        actions[counter] = action;
+        type[counter] = "T";
+        counter= counter + 1;
```

Comment:
Shouldn't there be a space before "="
```suggestion
        counter = counter + 1;
```